docs/docs/icechunk-for-git-users.md

```diff
@@ -60,7 +60,7 @@ session = repo.readonly_session("my-new-branch")
 session = repo.writable_session("my-new-branch")
 ```
 
-Once we have checked out a session, the [`store`](reference.md#icechunk.Session.store) method will return a [`Store`](reference.md#icechunk.Store) object that we can use to read and write data to the repository with `zarr`.
+Once we have checked out a session, the [`store`](reference.md#icechunk.Session.store) method will return a [`Store`](reference.md#icechunk.IcechunkStore) object that we can use to read and write data to the repository with `zarr`.
```

Comment:
This seems correct to me, but would be great to have someone verify

Reply:
yes correct.